pkg/reconciler/taskrun/taskrun.go

```diff
@@ -507,6 +544,15 @@ func (c *Reconciler) reconcile(ctx context.Context, tr *v1beta1.TaskRun, rtr *re
 	}
 
 	if podconvert.SidecarsReady(pod.Status) {
+		if spireEnabled {
+			// TTL for the entry is in seconds
+			ttl := time.Duration(config.FromContextOrDefaults(ctx).Defaults.DefaultTimeoutMinutes) * time.Minute
```

Comment:
What's the purpose of this TTL? Does it need to match the pod timeout?

What happens if the pod/task timeout != the default?